sqlmesh/dbt/model.py

```diff
@@ -203,7 +199,8 @@ def model_kind(self, target: TargetConfig) -> ModelKind:
                 IncrementalUnmanagedKind
             )
             return IncrementalUnmanagedKind(
-                insert_overwrite=strategy in INCREMENTAL_BY_TIME_STRATEGIES
+                insert_overwrite=strategy in INCREMENTAL_BY_TIME_STRATEGIES,
+                forward_only=self.forward_only,
```

Comment:
Should you use `foward_only=incremental_kwargs["forward_only"]` in case it is defined in the `meta`?